### DISCUSSION TITLE
Support URL Databases

### DISCUSSION THREAD

**Discussion by ArashTheKID** [keepassxreboot/keepassxc#7984] [Ideas]
Hi,
it would be nice to be able to open not only **Local** databases, but databases over **URLs** as well (WebDAV).
Cheers and thx,
A.

**Comment by Stratop0w** [comment]
Hi,

I agree, wanted to migrate from original KeePass for Windows, but I need to wait until this missing feature is implemented (or other ways to access a DB that is stored somewhere multiple systems like PC + mobile + tablet can access the database from everywhere :)

Cheers and thanks for this nice app :)
